### DISCUSSION TITLE
TSSs used for distance calculation in LinkPeaks

### DISCUSSION THREAD

**Discussion by gegenhu** [stuart-lab/signac#801] [Ideas]
Hi,

Is it possible to specify which transcript TSSs are used for calculating the distance parameter in LinkPeaks (i.e., all transcripts vs. longest transcript)? I'm working with a set of genes that have isoforms of highly variable lengths (100Kb vs. 900Kb, for instance). Using only the TSS of the longest transcript often fails to detect peaks that are within the selected distance of the shorter transcript.

Thanks,
Bruno

**Comment by timoast** [comment]
You can use the `gene.coords` parameter to manually supply the coordinates of genes or transcripts to test
